samples/getting_started/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+// Update to your credentials
+cloudinary.config({
+    cloud_name: '<cloud_name>',
+    api_key: '<api_key>',
+    api_secret: '<api_secret>'
+});
+
+// Ensure that you're registered to auto tagging https://cloudinary.com/documentation/google_auto_tagging_addon
+// Ensure that you're registered to background removal https://cloudinary.com/documentation/cloudinary_ai_background_removal_addon
+//Upload an image with auto tag
+//apply eager transformation
+cloudinary.uploader.upload(
+    "https://res.cloudinary.com/demo/image/upload/v1707306308/cld-docs-hp/walking_woman",
+    {
+        public_id: "walking_woman",
+        categorization: "google_tagging",
+        eager: [{effect: "background_removal"}]
+    })
+    .then((result)=>{
+        console.log(result.info.categorization.google_tagging)}).catch((error)=> {console.log(error)});
+
+// log in to you cloudinary account. Navigate to the newly uploaded `walking_woman` image. Select open on the image
+// and navigate to the Metadata tab. Select Auto tags, select plus on the Overcoat tag to tag your image.
```

Comment:
```suggestion
```

Reply:
We shouldn't need these comments anymore after adding the auto-tagging param to the upload command.